src/gui/gtk.c

```diff
@@ -1184,6 +1184,41 @@ int dt_gui_gtk_init(dt_gui_gtk_t *gui)
   c[DT_GUI_COLOR_BRUSH_TRACE] = (GdkRGBA){ 0., 0., 0., 0.8 };
   gtk_style_context_lookup_color(ctx, "brush_trace", &c[DT_GUI_COLOR_BRUSH_TRACE]);
 
+  c[DT_GUI_COLOR_THUMBNAIL_BG] = (GdkRGBA){ 0.4, 0.4, 0.4, 1.0 };
```

Comment:
I don't think this is correct, it may or may not work as gtk_style_context_lookup_color() does not say that the color is not changed if not found. In dt we currently do:
```
if(!gtk_style_context_lookup_color(ctx, "thumbnail_bg_color", &c[DT_GUI_COLOR_THUMBNAIL_BG]))
   c[DT_GUI_COLOR_THUMBNAIL_BG] = (GdkRGBA){ 0.4, 0.4, 0.4, 1.0 };
```

Reply:
In practice it does work at least for me, but why not be more strict. We'll need to update the similar code right above too (I'm guilty for it too). Actually, I think it's time to introduce a helper function to make this a one-liner. Will do later.

Reply:
Done.